### PR TITLE
Clarify CUE_FS_ROOT and fix for macOS.

### LINKED ISSUE #148: Update Getting Started guide with options for specifying a log directory
macOS Catalina makes `/` a [read-only filesystem](https://support.apple.com/en-us/HT210650), preventing new subdirectories from being created there. This means that our default log directory of `/shots` can't be created on macOS, causing the existing Getting Started guide to be hard to follow for macOS users.

We should update the Getting Started guide to specify a custom log directory for any components that need it. This includes:
- Cuebot
  - Use the `CUE_FRAME_LOG_DIR` env var to override this -- can be passed to the Docker container with the `-e` flag. This setting should then become the default log directory, applied to any new jobs that get submitted.
- RQD
  - Update the [Deploying RQD page](https://www.opencue.io/docs/getting-started/deploying-rqd/#before-you-begin) with new `CUE_FS_ROOT` setting, so the log directory gets shared to the host machine.

I think that should be all, but we'll need to test it more and possibly there are some other doc pages that will need to be updated.

Though this fixes a macOS-specific problem, I think providing an example for overriding the log directory will be useful for other platforms as well -- it's probably the first thing that sysadmins will need to figure out how to change once they move from a Getting Started-type deployment to a real production deployment.

### PR DESCRIPTION
**Pull request by bcipriano**
Fixes #148 

- Specify a new location for logs to be written. The default locations of `/shots` no longer works on macOS.
  - This also has the side effect of demonstrating this feature for the user, which is extremely helpful when figuring out how to create an actual production deployment.
- Maintaining the concept of `CUE_FS_ROOT`, write logs into a `logs/` subdirectory. In the event a user uses their actual NFS for this, we don't want to pollute that directory with a bunch of project folders. IMO it's obvious enough how you could adapt your actual project directory structure to match this, once you see the directory structure that gets created in `logs/`.

### CODE REVIEW

**Comment by bcipriano** [comment]
Note that this depends on https://github.com/AcademySoftwareFoundation/OpenCue/pull/627 which adds support for `--log.frame-log-root`.

File: content/docs/Getting started/deploying-cuebot.md [hunk 1]
```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
...
+    filesystem such as NFS is often used for this purpose. For the purposes of
+    this tutorial, you can use any directory you have write access to.
+
+    ```shell
+    export CUE_FS_ROOT="$HOME/opencue-tutorial"
+    mkdir -p "$CUE_FS_ROOT"
+    ```
+
+    If you plan to run [CueGUI](/docs/getting-started/installing-cuegui) and
+    [CueSubmit](/docs/getting-started/installing-cuesubmit) on the same host,
```
Comment (sharifsalah) [inline content/docs/Getting started/deploying-cuebot.md]: For the sake of people who are very new to all the terms in the project, it might be helpful to be a bit more explicit and say something like:

If you plan to run CueGUI and CueSubmit on the same host as CueBot, then all three components can use the local filesystem on that host.

How does that sound?

File: content/docs/Getting started/deploying-cuebot.md [hunk 2]
```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
     to the database. This configuration is outside of the scope of this
     guide.{{% /alert %}}
 
+1.  Cuebot needs to know the location of the filesystem where render assets
+    are stored and log files are written. In a large-scale deployment a shared
+    filesystem such as NFS is often used for this purpose. For the purposes of
+    this tutorial, you can use any directory you have write access to.
```
Comment (sharifsalah) [inline content/docs/Getting started/deploying-cuebot.md]: Nit: as tutorials are a specific type of doc, I recommend rewording to:

For the purposes of this guide, you can...

File: content/docs/Getting started/deploying-cuebot.md [hunk 3]
```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
...
+
+    ```shell
+    export CUE_FS_ROOT="$HOME/opencue-tutorial"
```
Comment (sharifsalah) [inline content/docs/Getting started/deploying-cuebot.md]: Here and in the RQD installation guide, I find the use of 'tutorial' a little unusual. As an admin, I would expect to be following this install guide to set up my system for testing or production. Consider `/opencue-testing` or `/opencue-demo` ?

Part of my rationale is that it might one day be good to have a proper end-to-end tutorial :)

File: content/docs/Getting started/deploying-cuebot.md (hunk 1 above)
Comment (bcipriano) [reply content/docs/Getting started/deploying-cuebot.md]: Sounds good -- done.

File: content/docs/Getting started/deploying-cuebot.md (hunk 2 above)
Comment (bcipriano) [reply content/docs/Getting started/deploying-cuebot.md]: Done.

File: content/docs/Getting started/deploying-cuebot.md (hunk 3 above)
Comment (bcipriano) [reply content/docs/Getting started/deploying-cuebot.md]: Makes sense, changed to `opencue-demo`.

**Comment by bcipriano** [comment]
Let's hold on merging this until https://github.com/AcademySoftwareFoundation/OpenCue/pull/627 is merged.

**Comment by bcipriano** [comment]
AcademySoftwareFoundation/OpenCue#627 is merged now, and is included in today's OpenCue release.

I QA'd the release by following the version of the Getting Started guide contained in this PR, and all looks good. Merging.
